BugTriaging.md

```diff
@@ -0,0 +1,84 @@
+# Teku Bug Triage Process
+
+In Teku, we have a well-defined policy regarding bug categorization. 
+
+#### Deciding Priority
+When deciding how to respond to a bug, consider the two variables of probability and severity, and apply them to a risk matrix to determine the priority and corresponding action.
+
+#### Probability
+How often actual customers are likely to encounter the issue during their use of the product?
+
+A feature rarely used, with a high likelihood of failure, may have a lower overall probability than a popular feature with a low likelihood of failure (due to a high number of customers affected).
+
+| Frequent   | Will occur several times in the lifetime of the product.          |
```

Comment:
The formatting on this table isn't quite right - the first line is being used as a header.  You can embed most HTML in markdown which may be the easiest way to get the tables to format correctly.

Reply:
Good catch. Will fix.